pages/fundamentals/introduction.html

```diff
@@ -1,190 +1,185 @@
+	다른 대중적인 프로그래밍 언어와 유사합니다.
+	이를 통해 Go 프로그래밍을 쉽게 익힐 수 있습니다.
 </li>
 <li>
-	Go comes with a great set of standard code packages which provide all kinds of
-	common functionalities. Most of the packages are cross-platform.
+	Go는 모든 종류의 공통 기능을 제공하는 훌륭한 표준 코드 패키지 세트와 함께 제공됩니다.
+	대부분의 패키지는 크로스 플랫폼입니다.
 </li>
 <li>
-	Go also has an active community, and there are
-	<a href="https://github.com/avelino/awesome-go">plenty of
-	high quality third party Go packages and projects</a> to import and use.
+	Go에는 활발한 커뮤니티도 있고
+	<a href="https://github.com/avelino/awesome-go">많은
+	고품질의 타사 Go 패키지 및 프로젝트</a>를 가져와서 사용할 수 있습니다.
 </li>
 </ul>
 
 </div>
 
 <p>
-Go programmers are often called gophers.
+Go 프로그래머는 고퍼들(Gophers)이라고 불립니다.
 </p>
 
 <p>
-In fact, although Go is a compiled and static typed programming language,
-Go also has many features which are usually only available in dynamic script languages.
-It is hard to combine these two kinds into one language, but Go did it.
-In other words, Go owns both the strictness of static languages
-and the flexibility of dynamic languages.
-I can't say there are not any compromises between the two,
-but the effect of the compromises is much weaker than the benefits of the combination in Go.
+실제로 GO는 컴파일되고 정적 타입의 프로그래밍 언어이지만 GO에서는 일반적으로 동적 스크립트 언어에서만 사용할 수 있는 많은 기능도 있습니다.
+두 종류를 하나의 언어로 합치는 것은 어렵지만 GO에서는 해냈습니다.
+즉, Go는 정적 언어의 엄격함과 동적 언어의 유연성을 모두 가지고 있습니다. 둘 사이에 문제가 없었다고 말할 수는 없지만 Go에서의 이점 보다 훨씬 덜합니다.
 </p>
 
 <p>
-Readability is an important factor which affects the design of Go heavily.
-It is not hard for a gopher to understand the Go code written by other gophers.
+Go의 디자인에서 가독성은 크게 영향을 미치는 중요한 요소입니다.
+고퍼가 다른 고퍼가 작성한 Go 코드를 이해하는 것은 어렵지 않습니다.
 </p>
 
 <p>
-Currently, the most popular Go compiler is written in Go and maintained by the Go design team.
-Later we shall call it the standard Go compiler, or <code>gc</code>
-(an abbreviation for Go compiler, not for garbage collection GC).
-The Go design team also maintains a second Go compiler, <code>gccgo</code>.
-Nowadays it's use is less popular than <code>gc</code>, but it always serves as a reference,
-and both compilers are in active development.
-As of now the Go team focuses on the improvement of <code>gc</code>.
+현재 가장 인기 있는 Go 컴파일러는 Go로 작성되었으며 Go 디자인 팀에서 유지 관리합니다.
+나중에 표준 Go 컴파일러 또는 <code>gc</code>라고 부를 것입니다.
+(가비지 컬렉션 GC가 아닌 Go 컴파일러의 약자).
+Go 디자인 팀은 두 번째 Go 컴파일러인 <code>gccgo</code>도 유지 관리합니다.
+요즘에는 <code>gc</code>보다 사용 빈도가 낮지만 항상 참조용으로 사용됩니다.
+두 컴파일러 모두 활발하게 개발 중입니다.
+현재 Go 팀은 <code>gc</code>의 개선에 집중하고 있습니다.
 </p>
 
 <p>
-<code>gc</code> is provided in Go Toolchain
-(a set of tools for Go development maintained by Go team).
-Go Toolchain 1.0 was released in March, 2012.
-The version of Go is consistent with the version of Go Toolchain.
-There were/are two major versions released each year.
+<code>gc</code>는 Go Toolchain에서 제공됩니다.
+(Go 팀에서 관리하는 Go 개발용 도구 세트).
+Go Toolchain 1.0은 2012년 3월에 출시되었습니다.
+Go 버전은 Go Toolchain 버전과 일치합니다.
+매년 두 개의 주요 버전이 릴리즈되었습니다.

```

Comment:
이거 원문 대체 왜 저런 식으로 써 놨을까요
그냥 현재형으로 쓰는 게 나을 것 같습니다. 과거형이면 더 이상 업데이트가 안 되는 것 같은 뉘앙스를 풍기네요.

```suggestion
매년 두 개의 주요 버전이 릴리즈됩니다.
```

Reply:
수정했습니다